model/metrics/gen-ai.yaml

```diff
@@ -43,3 +43,27 @@ groups:
           The `error.type` SHOULD match the error code returned by the Generative AI provider or the client library,
           the canonical name of exception that occurred, or another low-cardinality error identifier.
           Instrumentations SHOULD document the list of errors they report.
+  - id: metric.gen_ai.server.latency.request_duration
+    type: metric
+    metric_name: gen_ai.server.latency.request_duration
+    brief: 'Time (end to end latency) to complete a request'
+    instrument: histogram
+    unit: "s"
+    stability: experimental
+    extends: metric_attributes.gen_ai
```

Comment:
this one does not include `error.type` which would also apply to server metrics.

You might want to create a new attribute group and extend it, e.g.:

```yaml
  - id: metric_attributes.gen_ai.server
    type: attribute_group
    brief: 'This group describes GenAI server metrics attributes' # not rendered 
    extends: metric_attributes.gen_ai
    attributes:
      - ref: error.type
        requirement_level:
          conditionally_required: "if the operation ended in an error"
        note: | 
          The `error.type` SHOULD match the error code returned by the Generative AI service,
          the canonical name of exception that occurred, or another low-cardinality error identifier.
          Instrumentations SHOULD document the list of errors they report. 
```

It can also be reused on client duration metric which now references attribute directly, but then we'd need to make note applicable to both.

Reply:
The metrics on the server side unlike the client metrics are not at a single request level, but are cumulative for all the requests that have been processed by the server till that point in time which are then bucketed accordingly and reported. So, I'm not sure if `error.type` is needed for it since individual requests might fail, but the cumulative metric will report data on all the requests processed so far.

Reply:
the client metrics are also aggregates, but they are aggregated within one combination of attributes.

If we don't report error.type, it's not possible to distinguish successful responses from non-successful, get error rate, the duration would be skewed, etc. 
So it's be hard to use such metric. 

Are there some blockers that does not allow servers to use error type as a dimension?

Reply:
Created a new attribute group as discussed and used it for request duration.